packages/jest-preset-mc-app/setup-test-framework.js

```diff
@@ -13,7 +13,12 @@ const loadConfig = require('./load-config');
 
 const jestConfig = loadConfig();
 
-Enzyme.configure({ adapter: new Adapter(), disableLifecycleMethods: true });
+Enzyme.configure({
+  adapter: jestConfig.EnzymeAdapter
```

Comment:
So we default again the the old version and offer an option to pass in any adapter you want. An alternative is a config string e.g. v17 and v16 but then both dependencies need to reside here and it gives the impression we support any version.